tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -505,6 +526,19 @@ private function getDatasetsForStock(): array
         $expectedOutputData['stock']['availability']['available_later_label'] = $localizedValues;
         $expectedOutputData['stock']['availability']['available_date'] = '1969-07-20';
 
+        $expectedOutputData['stock']['quantities']['stock_movements'] = [
+            [
+                'date_add' => '2021-05-24 15:24:32',
+                'employee' => 'Paul Atreide',
```

Comment:
Looks like you are planning to watch Dune soon in theaters? 😄

Reply:
Maybe 😅 I just wan to finish (re)reading the book first